lib/modules/manager/helmv3/readme.md

```diff
@@ -64,3 +64,14 @@ For this you use a custom `hostRules` array.
   ],
 }
 ```
+
+### SubChart archives
```

Comment:
```suggestion
### Subchart archives
```

The helm docs call it `subchart` or `subcharts` [^helm]. So let's use their terms in our docs. 😉 

[^helm]: https://helm.sh/docs/chart_template_guide/subcharts_and_globals/